src/components/LoginButton.jsx

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from 'react';
+import firebase from 'firebase';
+import Button from '@material-ui/core/Button';
+import LoginDialog from './LoginDialog';
+
+export default function LoginButton() {
+  const { currentUser } = firebase.auth(); // determines how to initialize isSignedIn
```

Comment:
Hm, made the change and it cause a lint error and its fixable via lint but convert it to `const { currentUser } = firebase.auth();`

Reply:
What was the lint error?

Reply:
If it was a `destructuring-props` issue then I would just use `firebase.auth().currentUser` where `currentUser` is used since It's only used in one place

Reply:
Please don't mark the conversation resolved unless the comment was actually addressed. In this case, there's still open questions.

Reply:
7:9  error  Use object destructuring  prefer-destructuring